ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -20,6 +20,7 @@ nginx_sites:
    location2:
     name: "/static"
     alias: "{{ nginx_static_home }}"
+    add_header: "Access-Control-Allow-Origin *;"
```

Comment:
thanks, was hand editting the nginx conf and hadn't given it a go yet. will definitely ensure that it runs on staging before deploying to prod